.github/ISSUE_TEMPLATE/Bug_Report.md

```diff
@@ -0,0 +1,66 @@
+---
+name: 🐛 Bug Report
+about: If something isn't working as expected 🤔.
+labels: bug/documentation/question
```

Comment:
labels: bug
Is there a way to do optional labels? I am not sure but if yes, documentation and question should move there.